src/server/pachyderm_test.go

```diff
@@ -6097,13 +6097,20 @@ func TestCronPipeline(t *testing.T) {
 					})
 				}))
 				listCommitClient, err := c.PfsAPIClient.ListCommit(c.Ctx(), &pfs.ListCommitRequest{
-					Repo:       client.NewRepo(repo),
-					OriginKind: pfs.OriginKind_USER,
+					Repo:       client.NewRepo(pipeline8),
```

Comment:
after some archaeology, I think this whole list commit part can be deleted, and we should go back to checking the file count in the downstream repo (could just wait on that commit instead of `WaitCommitSetAll`)